pkg/alpine/mirrors/3.9

```diff
@@ -67,6 +73,7 @@ patch
 perl
 pixman-dev
 protobuf-dev
+py-docutils
```

Comment:
This can be avoided too if you pass --enable-generate-man-pages=no to configure in setup-rsyslog.sh

Reply:
done